sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb

```diff
@@ -284,16 +268,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"Endpoint name: \" + predictor.endpoint)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import sagemaker\n",
+    "print(\"Endpoint name: \" + predictor.endpoint_name)\n",
     "predictor.delete_endpoint()"
    ]
   }
```

Comment:
Some folks are still using Python SDK v<2. Since we upgraded above, let's revert back to their original version. Something like: 

In the beginning of the notebook: 
```
# Get original version:
import sagemaker
original_version = sagemaker.__version__
# update sagemaker
if original_version.split('.')[0] == '1':
    !~/anaconda3/envs/mxnet_p36/bin/pip install --upgrade sagemaker
.
.
.
```
At the end of the notebook, add:
```
.
.
.
# Revert back to their original Python SDK
s = 'sagemaker=={}'.format(original_version)
print("Rolling back to... {}".format(s))
%pip install {s}
import sagemaker
print("{} installed!".format(sagemaker.__version__))
```